src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: $card-paddings;
+  width: $card-width;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-left: 3px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-general-text;
+    margin-bottom: 4px;
+  }
+
+  &__code-item {
+    color: $color-secondary-text;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__feedback-quantity {
+    line-height: 14px;
+    font-size: 10px;
+    color: $color-general-text;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary-text;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-general-text;
+  }
+
+  &__button-buy {
+    display: block;
+    height: 40px;
+    line-height: 40px;
+    text-align: center;
+    text-decoration: none;
+    background-color: $color-general;
+    border: 1px solid $color-general;
+    border-radius: 5px;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+
+    &:hover {
+      background-color: #fff;
```

Comment:
Would be great to create variables for all the colors, so then you can change them in one place